docs-website/docs/overview/sdk.md

```diff
@@ -2,3 +2,92 @@
 sidebar_position: 3
 title: SDK
 ---
+
+# typesafe-i18n
+An opinionated, fully type-safe, lightweight localization library for TypeScript and JavaScript projects with no external dependencies.
+
+## Installation
+### 1.1. Install typesafe-i18n, the importer and concurrently
+
+
+```bash
+npm i typesafe-i18n
+npm i @inlang/typesafe-i18n-importer
+npm i concurrently --save-dev
+```
+
+### 1.2. Create the .typesafe-i18n.json config file
+
+- `adapter` specifies that the generates i18n files should be react compatible.
+
+```js title="typesafe-i18n.json"
+{
+  "$schema": "https://unpkg.com/typesafe-i18n@2.40.1/schema/typesafe-i18n.json",
+  "adapter": "svelte"
+}
+```
+
+### 1.3. Create the `inlang.config.json` file.
+
+- `projectId`: create a project at [inlang](https://app.inlang.dev) and copy the project id
+- `wrappingPattern`: defines how a key (keyname) should be wrapped when creating a key with the [inlang
+  VSCode extension](https://marketplace.visualstudio.com/items?itemName=inlang.vscode-extension). For Svelte Kit it's
+  "$LL.keyname()".
+
+```js title="inlang.project.json"
+{
+  "projectId": "YOUR PROJECT ID",
+  "vsCodeExtension": {
+    "wrappingPattern": "$LL.keyname()"
+  }
+}
+```
+
+### 1.4. Adjust the build script
+
+The SDK (typesafe-i18n & the inlang typesafe importer) run as background processes during development to constantly fetch updated translations from the dashboard and generate corresponding types. Since the processes should run simultaneously next to the regular development process (`npm run dev`), we adjust the dev script in the `package.json` to run the regular dev script, the SDK and the importer in parallel with the help of [concurrently](https://www.npmjs.com/package/concurrently).
```

Comment:
Nice section. Oh wait, it was written by me. In this article, this section should move to the top.